CHANGELOG.md

```diff
@@ -1,55 +1,65 @@
-# Release notes
```

Comment:
The idea was to have release notes in the documentation, as then you can use sphinx to actually link to class definitions etc. Maybe a CHANGELOG.md that links to the documentation file?

Reply:
But then I am wondering, why it was written as `.md`?
But I can revert that.

Reply:
And a note at this point: This file was not even showing up in the doc (See: https://geostat-framework.readthedocs.io/projects/pykrige/en/stable/)

In the release notes, I think it is also better to link to PRs and commits to show changes. But being able to reference classes etc. sounds good as well.

Reply:
> But then I am wondering, why it was written as .md?

It was from a time I though it was a good idea. sphinx can parse markdown, but I think converting it to rst would be easier.

Reply:
Keep it for the moment.